script.js

```diff
@@ -23,10 +35,16 @@ const searchCity = () => {
     })
 }
 
+/**
+ * Gets the weather data of the city and returns the promise.
+ * @param {String} city holds the city name, of which weather data we want to get.
+ * @returns Promise if successfull or error if something wrong happened.
+ */
 const getWeather = (city) => {
     const options = {
         method: 'GET',
         headers: {
+            // The old API Key has been deleted successfully.
```

Comment:
Good you have confirmed that, old key has been deleted.  
Have you generated a new key to access the API

Reply:
Yes, I have generated a new key to access the API.